spec/views/articles/_article.html.erb_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+require 'spec_helper'
+
+describe "views/articles/_article.html.erb" do
+  context "with user" do
+    let(:user)  { build :user }
```

Comment:
Ahh sorry - ignore this file. It was when I was following someone's tutorial to dip my toes into testing before you started helping me.